app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -130,6 +130,15 @@ def sync
       respond_for_async :resource => task
     end
 
+    api :POST, "/repositories/:id/export", N_("Export a repository")
+    param :id, :identifier, :required => true, :desc => N_("repository ID")
+    param :since, String, :desc => N_("ISO 8601 date of last export (ex: 2010-01-01T12:00:00), useful for exporting deltas.")
```

Comment:
I added date format to gutterball API and agree that we should be consistent across the product: foreman, katello, plugins, etc.

https://github.com/Katello/foreman-gutterball/blob/master/app/controllers/foreman_gutterball/api/v2/content_reports_controller.rb#L13